composer.json

```diff
@@ -283,9 +283,6 @@
             },
             "drupal/site_alert": {
                 "Add ability to dismiss alert - https://www.drupal.org/project/site_alert/issues/3156557#comment-14930684": "https://www.drupal.org/files/issues/2023-02-19/3156557-67.patch"
-            },
-            "drupal/ckeditor_templates": {
-                "CKEditor 5 support for Content Templates - https://www.drupal.org/project/ckeditor_templates/issues/3273358#comment-14963779": "./patches/ckeditor_templates/ckeditor_templates_4.patch"
```

Comment:
I couldn't get this patch to apply to content-vic. Pulled it out for now, will re-add before I merge.